test/jira-tests.js

```diff
@@ -40,6 +41,24 @@ describe('Jira API Tests', () => {
       expect(jira.baseOptions.auth).to.be.undefined;
     });
 
+    it('Constructor with oauth credentials', () => {
+      const options = getOptions({ oauth: {
+        consumer_key: 'consumer',
+        consumer_secret: 'consumer_secret',
+        access_token: 'token',
+        access_token_secret: 'token_secret'
+      } });
```

Comment:
This sort of tabbing is slightly inconsistent with the remainder of the file.  Not trying to be nitpicky or difficult, I only bring this up because I nearly missed the `oauth: {}` scope when reading this over.

```javascript
const options = getOptions({ 
  oauth: {
    consumer_key: 'consumer',
    consumer_secret: 'consumer_secret',
    access_token: 'token',
    access_token_secret: 'token_secret'
  } 
});
```